src/main/resources/changesets/changelog-consent-71.0.xml

```diff
@@ -0,0 +1,26 @@
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="71.0" author="grushton">
+        <validCheckSum>ANY</validCheckSum>
+
+        <addColumn tableName="dacuser">
+            <column name="era_commons_id" type="varchar(255)"/>
+        </addColumn>
+
+        <sql>
```

Comment:
Didn't realize you could do this with the changelogs. This makes simple backfills easy to do.

Reply:
Re-reading this, I think the query needs to be tested on each of the databases. It could be the case that the subquery returns multiple rows for a user id. Not likely, but certainly possible given the query.